package.json

```diff
@@ -15,7 +15,8 @@
     "serve": "npm run prebuild && vue-cli-service serve",
     "prebuild": "node update-embedded-version.js",
     "build": "npm run prebuild && vue-cli-service build",
-    "lint": "vue-cli-service lint"
+    "lint": "vue-cli-service lint",
+    "sbom": "cyclonedx-npm --output-file sbom.json"
```

Comment:
Any reason why this was added? We already produce an SBOM with webpack.

Reply:
Nop, no reason at all. It's gone now :-)